src/main.py

```diff
@@ -13,6 +13,23 @@
 from game_info import TITLE
 from window_size import WINDOW_SIZE
 
+
+def compile_auto_script():
```

Comment:
transformed into a function so that pylint doesn't complain about _constants_ without uppercase